src/particles/elements/Drift.H

```diff
@@ -92,6 +107,8 @@ namespace impactx
             py = pyout;
             pt = ptout;
 
+            // undo shift due to alignment errors of the element
+            shift_out(x, y, px, py);
```

Comment:
``
```suggestion
            shift_out(x, y, px, py);
            p.pos(RealAoS::x) = x;
            p.pos(RealAoS::y) = y;
```

Reply:
In the preceding lines, we also need to replace

`p.pos(RealAoS::x) = x + slice_ds * px;`

with

`x = x + slice_ds*px;`

with a corresponding change for y.

Reply:
Great catch!

This might work as well:
```suggestion
            shift_out(p.pos(RealAoS::x), p.pos(RealAoS::y), px, py);
```

Reply:
Fix pushed :)